tools/tools.go

```diff
@@ -23,6 +23,7 @@ import (
 	_ "github.com/brancz/gojsontoyaml"
 	_ "github.com/campoy/embedmd"
 	_ "github.com/google/go-jsonnet/cmd/jsonnet"
+	_ "github.com/hairyhenderson/gomplate/v3"
```

Comment:
```suggestion
	_ "github.com/hairyhenderson/gomplate/v4"
```

I see `v4` is out, can we use that instead?

Reply:
v4 still has no official release, just pre-releases. I'd prefer to stay on v3 for now.